eZ/Publish/Core/Persistence/Cache/Tests/AbstractBaseHandlerTest.php

```diff
@@ -65,24 +69,25 @@ final protected function setUp()
         $this->cacheMock = $this->createMock(TransactionalInMemoryCacheAdapter::class);
         $this->loggerMock = $this->createMock(PersistenceLogger::class);
         $this->inMemoryMock = $this->createMock(InMemoryCache::class);
+        $this->tagGenerator = new TagGenerator();
```

Comment:
For some reason, simple mocking resulted in many tests failing so decided to just create an instance of `TagGenerator`.